platforms/common/src/main/resources/assets/dynamic_fps/lang/de_at.json

```diff
@@ -33,10 +69,20 @@
     "config.dynamic_fps.run_garbage_collector": "GC auslösen",
     "config.dynamic_fps.run_garbage_collector_tooltip": "Ungenutzen Arbeitsspeicher frei machen, wenn dieser Status aktiviert wird",
 
+    "config.dynamic_fps.download_natives": "Bibliotheken Herunterladen",
+    "config.dynamic_fps.download_natives_description_0": "Nutzung der Akku-Features erfordert eine weitere Bibliothek.",
+    "config.dynamic_fps.download_natives_description_1": "Stelle ein, ob die Mod diese für dich herunterladen darf.",
```

Comment:
This is a personal thing, but I've always said "der Mod". Maybe "die Mod" is more widespread though!

Reply:
I also say "der Mod" in casual conversation usually, but since it is a shorthand for "die Modifikation" I think this is actually correct.